app/models/planning_application.rb

```diff
@@ -935,4 +952,12 @@ def filter_documents_for_tab(documents, tab)
 
     documents.select { |document| (tags & document.tags).any? }
   end
+
+  def modify_expiry_date
+    if environment_impact_assessment
+      self.expiry_date += 56.days unless environment_impact_assessment_was
```

Comment:
What is `environment_impact_assessment_was`?

Reply:
What the previous value of it was before the update. 
It will cover both false and nil cases. It effectively says, "if the previous value was not true (i.e., it was either false or nil), then update the expiry date."